config/README.rst

```diff
@@ -658,6 +658,22 @@ Hints
 * Call ``bin/check-python-versions <path-to-package> -h`` to see how to fix
   version mismatches in the *lint* tox environment.
 
+Updating to the currently supported Python versions
+---------------------------------------------------
+
+There is `update-python-support.py` which can be used to update a repository to
+the currently supported Python versions as defined in ``shared/package.py``.
+
+Usage
++++++
+
+To update a repository to the currently supported Python versions call::
+
+    $ bin/python update-python-support.py <path-to-package>
+
+It supports a parameter ``--interactive`` to not automatically commiting the
```

Comment:
```suggestion
It supports a parameter ``--interactive`` to not automatically commit the
```

Reply:
I'd really like an alternate spelling `--no-commit` for this flag so it is the same as for the main configure script, some way to make all changes but not do a `git commit`.